README.md

```diff
@@ -88,7 +88,7 @@ When we make an official release, we will create a unique git tag containing the
 
 ## SDK Architecture
 
-At the heart of our SDK is, what we refer to as, [Azure Core](sdk/core/core). This code defines several data types and functions for use by the client libraries that build on top of us such as an `Azure Storage Blob` client library and `IoT` client libraries. Here are some of the features that customers use directly:
+At the heart of our SDK is, what we refer to as, [Azure Core](sdk/core/core). This code defines several data types and functions for use by the client libraries that build on top of us such as an `Azure Storage Blob` client library and `IoT` [client libraries](https://github.com/Azure/azure-sdk-for-c/tree/master/sdk/iot). Here are some of the features that customers use directly:

```

Comment:
nit: I'm not a fan of mixing code style fonts and linking. I think both of these (Storage and IoT) should just be links.

Reply:
```suggestion
At the heart of our SDK is, what we refer to as, [Azure Core](sdk/core/core). This code defines several data types and functions for use by the client libraries that build on top of us such as an [Azure Storage Blob](https://github.com/Azure/azure-sdk-for-c/tree/master/sdk/storage) client library and [IoT client libraries](https://github.com/Azure/azure-sdk-for-c/tree/master/sdk/iot). Here are some of the features that customers use directly:
```